rust/src/messages.rs

```diff
@@ -39,11 +40,22 @@ pub(crate) struct RouterInternal {
     pub(crate) router_id: RouterId,
 }
 
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct WebRtcServerInternal {
+    #[serde(rename = "webRtcServerId")]
+    pub(crate) webrtc_server_id: WebRtcServerId,
+}
+
 #[derive(Debug, Serialize)]
 #[serde(rename_all = "camelCase")]
 pub(crate) struct TransportInternal {
     pub(crate) router_id: RouterId,
     pub(crate) transport_id: TransportId,
+    // TODO: This field should have really been in the `data`, not in `internal` data structure
```

Comment:
Why? When we call consume() the producerId is also included in internal object rather than in data. Everything that references internal/public ids is always given in the internal object.

Reply:
`internal` was always the thing that is included in every single request and is never optional, `webRtcServerId` seemed like something that only matters during creation of the transport and later can be inferred. There are no other methods except creation that require `webRtcServerId` field to be specified and for such cases we were putting one-time use fields into the `data` field.

So every single method before seemed to follow the same convention, while this one is an outlier. On that note, I think `internal` should be replaced completely with just an ID of corresponding entity (for transports it'll be transport ID). The reason for that is that worker already knows all the relationships and sending all the fields every time is a API that can potentially be used incorrectly, you can send one set of values in one method call and the other in the other method call, ideally we wouldn't even have an option of such possibility, making internal API more misuse-resistant.

Reply:
Channel messages are received by the Worker. If the Worker needs to know which Router a producerId belongs it should own a mapProduderToRouter (and so on). I think it's not that bad that we signal the entire tree in internal to avoid having those maps, but I see your point.

Whether webRtcTransportId should be part of data... I kinda agree. This is not for internal routing. I'm fine if w change this but that must also be done in other cases (I insist: when you call consume() the producerId is given in internal, it should also be given in data).